app/src/main/java/com/lastaoutdoor/lasta/ui/screen/moreinfo/MoreInfoScreen.kt

```diff
@@ -220,12 +227,16 @@ fun ViewOnMapButton(isMapDisplayed: MutableState<Boolean>) {
               Text(
                   LocalContext.current.getString(R.string.on_map),
                   style =
-                      TextStyle(
-                          fontSize = 16.sp,
-                          lineHeight = 24.sp,
-                          fontWeight = FontWeight(500),
-                          letterSpacing = 0.15.sp,
-                      ))
+                      when (Locale.getDefault().language) {

```

Comment:
On the text that was previously written, there was a custom letterspacing and I think that it would be better to either remove it or to put it also in your added condition to have more unified look.

What would be even better to avoid duplication of code is to have one textstyle for the two and chage only the fontsize with a condition

Reply:
even better I'll give you that !